js/src/bqplot.less

```diff
@@ -13,332 +13,301 @@
  * limitations under the License.
  */
 
+:root {
+    --bq-content-font-color: var(--jp-content-font-color1);
+    --bq-plotarea-background-fill: var(--jp-layout-color1);
+    --bq-background: var(--jp-layout-color0);
+
+    --bq-axislabel-font: 14px sans-serif;
+
+    --bq-axis-rect-opacity: 1.0;
+    --bq-axis-tick-stroke-width: 1.0;
+    --bq-axis-tick-opacity: 1.0;
+    --bq-axis-tick-text-font: 12px sans-serif;
+
+    --bq-mark-tooltip-border-radius : 5px;
+    --bq-mark-tooltip--webkit-user-select: none;
+    --bq-mark-tooltip--moz-user-select: none;
+
+    --bq-mainheading-font: 16px sans-serif;
+    --bq-mainheading-font-size: large;
+
+    --bq-world-map-path-stroke-width: 0.25px;
+
+    --bq-pie-polyline-stroke-width: 1px;
+    --bq-pie-polyline-fill: none;
+
+    --bq-network-node-element-stroke: #000;
+    --bq-network-node-element-stroke-width: 1.5px;
+
+    --bq-network-link-fill: none;
+    --bq-network-link-stroke: #666;
+    --bq-network-link-stroke-width: 1.5px;
+
+    --bq-network-node-label-pointer-events: none;
+    --bq-network-node-label-font: 12px sans-serif;
+
+    --bq-selector-fill: var(--jp-layout-color4);
+
+    --bq-mark-tooltip-border: 2px solid whitesmoke;
```

Comment:
Do we also want the tooltip styling to change with the theme?

Reply:
Yes we do. It actually does in this current state, I don't know what the mark_tooltip is.